### PR TITLE
Fix pcap replayer

### PR DESCRIPTION
**Pull request by benzea**
Stupid error in assert.

Also, looking at the code, I realised that control transfer playback cannot work right now. But, I am not sure what it should look like and don't have a good test case right now unfortunately (I hope to get some samples soon, if I do, it should be a quick fix).

### CODE REVIEW

**Comment by martinpitt** [comment]
@benzea (you are aware of the conflict?)

File: src/ioctl.vapi [hunk 1]
```diff
@@ -46,13 +46,13 @@ namespace Ioctl {
     [CCode (cheader_filename = "linux/usbdevice_fs.h")]
     public const int USBDEVFS_CAP_SUSPEND;
 
-    [CCode (cheader_filename = "asm-generic/ioctl.h")]
+    [CCode (cheader_filename = "asm/ioctl.h")]
```
Comment (martinpitt) [inline src/ioctl.vapi]: This commit is already in master, I suppose that's what it complains about.

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: src/ioctl.vapi (hunk 1 above)
Comment (benzea) [reply src/ioctl.vapi]: Oopsa, no, I forgot to rebase!

**Review by martinpitt — APPROVED** [review]
Danke! Want/need a release with this?

**Pull request merged**